commit ae56942c14740c2963222efdc36c667ab19555ef
Author: Kees Cook <kees@kernel.org>
Date:   Thu Jun 25 13:37:04 2020 -0700

    lkdtm: Make arch-specific tests always available
    
    I'd like arch-specific tests to XFAIL when on a mismatched architecture
    so that we can more easily compare test coverage across all systems.
    Lacking kernel configs or CPU features count as a FAIL, not an XFAIL.
    
    Additionally fixes a build failure under 32-bit UML.
    
    Fixes: b09511c253e5 ("lkdtm: Add a DOUBLE_FAULT crash type on x86")
    Fixes: cea23efb4de2 ("lkdtm/bugs: Make double-fault test always available")
    Fixes: 6cb6982f42cb ("lkdtm: arm64: test kernel pointer authentication")
    Signed-off-by: Kees Cook <keescook@chromium.org>
    Link: https://lore.kernel.org/r/20200625203704.317097-5-keescook@chromium.org
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/tools/testing/selftests/lkdtm/tests.txt b/tools/testing/selftests/lkdtm/tests.txt
index 92ca32143ae5..9d266e79c6a2 100644
--- a/tools/testing/selftests/lkdtm/tests.txt
+++ b/tools/testing/selftests/lkdtm/tests.txt
@@ -14,6 +14,7 @@ STACK_GUARD_PAGE_LEADING
 STACK_GUARD_PAGE_TRAILING
 UNSET_SMEP CR4 bits went missing
 DOUBLE_FAULT
+CORRUPT_PAC
 UNALIGNED_LOAD_STORE_WRITE
 #OVERWRITE_ALLOCATION Corrupts memory on failure
 #WRITE_AFTER_FREE Corrupts memory on failure